### DISCUSSION TITLE
Could not find available reservations

### DISCUSSION THREAD

**Discussion by rmize** [Alkaar/resy-booking-bot#94] [Q&A] [answered]
Doing some testing with tables I know are open to make sure I have bot working correctly. Getting the following everytime when there are tables open at times listed. Anyone else having issue or am I making basic mistake in config?

[INFO ] 2023-03-05 15:07:16.687-05:00 ResyBookingBot$:16 - Starting Resy Booking Bot
[INFO ] 2023-03-05 15:07:17.470-05:00 ResyBookingBot$:45 - Next snipe time: 2023-03-05T15:08:00.000-05:00
[INFO ] 2023-03-05 15:07:17.470-05:00 ResyBookingBot$:46 - Sleeping for 0 hours, 0 minutes, and 40 seconds
[INFO ] 2023-03-05 15:07:58.031-05:00 ResyBookingWorkflow:18 - Taking the shot...
[INFO ] 2023-03-05 15:07:58.032-05:00 ResyBookingWorkflow:19 - (҂‾ ▵‾)︻デ═一 (˚▽˚’!)/
[INFO ] 2023-03-05 15:07:58.032-05:00 ResyBookingWorkflow:20 - Attempting to snipe reservation
[INFO ] 2023-03-05 15:08:08.066-05:00 ResyClient:176 - Missed the shot!
[INFO ] 2023-03-05 15:08:08.066-05:00 ResyClient:177 - ┻━┻ ︵ \(°□°)/ ︵ ┻━┻
[INFO ] 2023-03-05 15:08:08.067-05:00 ResyClient:178 - Could not find any available reservations
[INFO ] 2023-03-05 15:08:08.067-05:00 ResyBookingBot$:53 - Shutting down Resy Booking Bot

**Config file (deleted / changed parts of api key / auth token:**
![image](https://user-images.githubusercontent.com/108693359/222983470-2190378c-50b7-4449-8493-85a00a77d31f.png)

**Comment by rmize** [comment]
Looking at log in console by changing info to debug. Saw someone in other thread said to take a look at anything starting with RGS. Not able to find anything with RGS which makes me think something is wrong with my config.

**Comment by rmize** [comment]
Only other thing I can think of is my auth token is incorrect. To confirm, it should be the whole sequence after "x-resy-auth-token:"? Not specific section as noticed long string seperated by "." a few times. 
![image](https://user-images.githubusercontent.com/108693359/222986640-d8d884b7-0c72-487f-825d-0bc1c56ebb37.png)

**Comment by dborges8** [comment]
I'm having the same issue. Had been using the bot successfully for some weeks

**Reply by Alkaar** [reply]
See the discussions section https://github.com/Alkaar/resy-booking-bot/discussions/91

You need to fix your payment info for the bot to work again. I just verified that it still works once you fix this.

**Comment by Alkaar** [comment, marked as answer]
@rmize looking at your config, did you put in the wrong year? Did you mean 2023?

**Reply by rmize** [reply]
I am dense, thank you sir

**Reply by Alkaar** [reply]
No worries. Happens to the best of us.
